include/deal.II/meshworker/scratch_data.h

```diff
@@ -520,6 +526,12 @@ namespace MeshWorker
     const GeneralDataStorage &
     get_general_data_storage() const;
 
+    /**
+     * Return a reference to the used mapping.
+     */
+    const Mapping<dim, spacedim> &
+    get_mapping() const;
+
```

Comment:
I moved this, because it was floating at the end of the public interface and gets slotted in after `get_general_data_storage()` in the docs.